### PR TITLE
Improve error handling and debug output

### PR DESCRIPTION
**Pull request by fredriv**

### CODE REVIEW

File: src/main/kotlin/no/kommune/oslo/maskinporten/cli/CreateClientCommand.kt [hunk 1]
```diff
@@ -13,8 +13,13 @@ class CreateClientCommand : AdminCommand(name = "client") {
     override fun run() {
         super.run()
 
-        val clientId = getAdminClient().createClient(name, description, scopes)
-        println("Client id: $clientId")
+        try {
+            val clientId = getAdminClient().createClient(name, description, scopes)
+            log.info("Created Maskinporten client with id $clientId")
+            println("""{"client_id": "$clientId"}""")
```
Comment (eirikbroen) [inline src/main/kotlin/no/kommune/oslo/maskinporten/cli/CreateClientCommand.kt:19]: redundant print statement?

File: src/main/kotlin/no/kommune/oslo/maskinporten/cli/CreateClientKeyCommand.kt [hunk 2]
```diff
@@ -39,10 +45,18 @@ class CreateClientKeyCommand : AdminCommand(name = "key") {
...
-        val clientKeyResponse = adminClient.registerClientKey(clientId, jwk)
-        log.debug("Client key response: $clientKeyResponse")
-
-        println("Wrote key with id $keyID to new keystore ${keyFile.path} with alias 'client-key'")
+        try {
+            adminClient.registerClientKey(clientId, jwk)
+
+            log.info("Registered new key with id $keyID for client $clientId")
+            log.info("Wrote key to keystore at ${keyFile.path} with alias 'client-key'")
+            println("""{"keystore_path": "${keyFile.path}", "key_id": "$keyID", "key_alias": "client-key", "client_id": "$clientId"}""")
```
Comment (eirikbroen) [inline src/main/kotlin/no/kommune/oslo/maskinporten/cli/CreateClientKeyCommand.kt:53]: redundant print statement or should also be a log.info statement?

File: src/main/kotlin/no/kommune/oslo/maskinporten/cli/ListClientKeysCommand.kt [hunk 3]
```diff
@@ -3,15 +3,23 @@ package no.kommune.oslo.maskinporten.cli
...
     private val clientId by option().help("Id of Maskinporten client to list keys for").required()
 
     override fun run() {
         super.run()
 
-        val clientKeys = getAdminClient().getClientKeys(clientId)
-        println(clientKeys)
+        try {
+            val clientKeys = getAdminClient().getClientKeys(clientId)
+            println(clientKeys)
```
Comment (eirikbroen) [inline src/main/kotlin/no/kommune/oslo/maskinporten/cli/ListClientKeysCommand.kt:16]: should be a log.info statement?

File: src/main/kotlin/no/kommune/oslo/maskinporten/cli/TokenCommand.kt [hunk 4]
```diff
@@ -24,12 +24,22 @@ class TokenCommand : BaseCommand() {
...
         val jwtConfig = getJwtConfig(issuer = clientId, consumerOrg, keyID)
         val wellKnownEndpoint = URL(config.getProperty("maskinporten.oidc.wellknown"))
 
-        val jwtAuthClient = JwtAuthClient(jwtConfig, wellKnownEndpoint)
-        val accessToken = jwtAuthClient.getAccessToken(setOf(scope))
-        println(om.writeValueAsString(accessToken))
+        try {
+            val jwtAuthClient = JwtAuthClient(jwtConfig, wellKnownEndpoint)
+            val accessToken = jwtAuthClient.getAccessToken(setOf(scope))
+            println(om.writeValueAsString(accessToken))
```
Comment (eirikbroen) [inline src/main/kotlin/no/kommune/oslo/maskinporten/cli/TokenCommand.kt:39]: should be a log.info statement?

**Review by eirikbroen — COMMENTED** [review]
Very nice work! All value adding changes in my opinion. Some minor questions on output, but go ahead an merge

File: src/main/kotlin/no/kommune/oslo/maskinporten/cli/CreateClientCommand.kt (hunk 1 above)
Comment (fredriv) [reply src/main/kotlin/no/kommune/oslo/maskinporten/cli/CreateClientCommand.kt:19]: The `log` output goes to `stderr` while the `println` goes to `stdout` and can be piped to other commands, e.g. `jq`.

**Review by yngvark — APPROVED** [review]
Nice improvements!

**Pull request merged**